fix(react): Address inline TOTP setup issues

LZoog opened the pull request:
Because:
* The inline TOTP setup flow is being blocked by customs on stage
* This flow can also encounter a "token already exists" error in some cases

This commit:
* Adds metricsContext to the request for customs
* Does not rerun the TOTP create request if a totp token already exists

fixes FXA-9435

---

You can access this flow in React by going to 123done and clicking "Sign in (2FA required)", adding the React experiment params, then either signing up for an account or signing into an account. You should be redirected back to signin with a cached view, then on click of signin, you should be taken to the inline TOTP setup page.

Vijay flagged in the comments of this ticket that the flow wasn't working as expected (note, QA's comment/gif is not applicable to this issue). I did some investigation and fixed a couple of issues I saw when going through the flow locally, though, it's worth noting the flow ID fix is a best guess to what I'm seeing on staging (request blocked for security reasons), I could not seem to trigger this state locally.

chenba commented:
Looking at the patch, I don't understand how it has anything to do with customs.

LZoog commented:
@chenba Do you know how to trigger a customs setup locally similar to what we have in prod? I changed `customsUrl` in `fxa-auth-server` to `http://localhost:7000/`, updated profile-server's, and updated GQL's in pm2, and couldn't trigger the state locally.

The reason I thought it might be blocked due to flowId is I noticed that [`createTotp` can take in `metricsContext`](https://github.com/mozilla/fxa/blob/670ae60eae0a0d39811b6225e01c83576331791e/packages/fxa-graphql-api/src/gql/account.resolver.ts#L160-L161), our `totp/create` endpoint takes it into the payload, and we do a [`customs.check` here](https://github.com/mozilla/fxa/blob/670ae60eae0a0d39811b6225e01c83576331791e/packages/fxa-auth-server/lib/routes/totp.js#L64-L65) without doing anything else with `metricsContext`. We're also sending this up in Backbone's version when using auth-client, but I wasn't totally sure this was the fix since `action` [is not `accountLogin`](https://github.com/mozilla/fxa/blob/670ae60eae0a0d39811b6225e01c83576331791e/packages/fxa-customs-server/lib/server.js#L300-L301) at this point. Settings `createTotp` is just using auth-client (... though I'm now seeing, that's without `metricsContext`), which I figured I might move to if this doesn't resolve the issue. The user's session is verified at this point and the GQL -> auth-server API issue is resolved, so I'm not sure what else it is. It's _possible_ also the `(!totp)` fix might do something for us, since I saw that query running 2-3 times sometimes. LMK if you have any ideas... If we have another dot release, I can ask for this to ride along.

LZoog commented:
Per convo in the weekly check-in I'm going to go ahead and merge this.

The Netlify CI stuff was apparently only on for 30 mins and was known to not work so ignoring those I'm seeing green CI across the board.
